awx/main/models/jobs.py

```diff
@@ -205,6 +205,11 @@ class Meta:
         app_label = 'main'
         ordering = ('name',)
 
+    job_type = models.CharField(
```

Comment:
I decided to *only* update `JobTemplate.job_type` and *not* `Job.job_type` for a few reasons:

1.  It might be nice to know (historically) that a job that ran _was_ a scan.
2.  It'll be a lot slower to update this column on large job counts.